src/applications/edu-benefits/10282/pages/applicantInformation/applicantGender.js

```diff
@@ -3,6 +3,41 @@ import fullSchema10282 from 'vets-json-schema/dist/22-10282-schema.json';
 
 const { gender } = fullSchema10282.properties;
 
+const gaGenderInfoHelpText = () => {
+  window.dataLayer.push({
+    event: 'edu-10282--form-IBM-SkillsBuild-program-gender-help-text-clicked',
+    'help-text-label':
+      'What to know before you decide to share your gender identity',
+  });
+};
+
+export const genderInfoHelpText = (
+  <va-additional-info
```

Comment:
add `class="vads-u-margin-top--2 vads-u-margin-bottom--2"` to va-additional-info component

Reply:
Done